libs/posthog_client_lite/src/lib.rs

```diff
@@ -39,6 +39,9 @@ pub struct LocalEvaluationResponse {
 
 #[derive(Deserialize)]
 pub struct LocalEvaluationFlag {
+    #[allow(dead_code)]
+    id: u64,
+    team_id: u64,
```

Comment:
Are these fields exists in both APIs?

If not, we will probably have `missing field` during the deserialization.